### PR TITLE
Add possibility to define a custom `rem()` function

### PR DESCRIPTION
**Pull request by kilobyte2007**
#495

### CODE REVIEW

**Comment by JosephusPaye** [comment]
Nice. This might be a good compromise since `@unless` doesn't work.

Could you also please update https://github.com/JosephusPaye/Keen-UI/blob/next/Customization.md to mention this?

**Comment by kilobyte2007** [comment]
Added a mention to the customization docs about this. Please let me know if it looks all right to you.

File: Customization.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ All other variables in [`keen-ui/src/styles/variables.scss`](.../src/styles/vari
 
 Some components also have component-specific variables that can be found in their source files. Those can be overridden as well.
 
+You can also override the `rem` function by defining a `rem-custom` function which would be used instead for calculating the size of components components. Please refer to the original function in [`keen-ui/src/styles/utils.scss`](.../src/styles/utils.scss).
```
Comment (JosephusPaye) [inline Customization.md:86]: Some suggestions:

```suggestion
You can also override the `rem` function by defining a `rem-custom` function to be used instead for calculating the size of components, as described below. Please refer to the original function in [`src/styles/utils.scss`](./src/styles/utils.scss).
```

**Comment by kilobyte2007** [comment]
Any updates on this?

**Comment by kilobyte2007** [comment]
Hey @JosephusPaye, any plans to merge this? I wish I could use this without having to specify the branch as the version in package.json. Thank you.

**Review by JosephusPaye — APPROVED** [review]

**Comment by JosephusPaye** [comment]
Thanks! Released in [v1.3.2](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.3.2)

**Pull request merged**